### PR TITLE
Make the Entry UI look a lot less weird

### PR DESCRIPTION
**Pull request by EasternBloxxer**
I am too tired to write anything to describe this pull request.
image

### CODE REVIEW

**Comment by EasternBloxxer** [comment]
this is what it looked like before
image

**Review by fxeP1 — APPROVED** [review]
LGTM

File: MainModule/Client/UI/Default/UserPanel.lua [hunk 1]
```diff
@@ -281,7 +285,7 @@ return function(data, env)
 					end
 				})
 
-				entryBox.BackgroundColor3 = entryBox.BackgroundColor3:lerp(Color3.new(1,1,1), 0.25)
+				--entryBox.BackgroundColor3 = entryBox.BackgroundColor3:lerp(Color3.new(1,1,1), 0.25)
```
Comment (P3tray) [inline MainModule/Client/UI/Default/UserPanel.lua:288]: just remove it completely, lazy bones

File: MainModule/Client/UI/Default/UserPanel.lua [hunk 2]
```diff
@@ -192,23 +192,26 @@ return function(data, env)
 				local entryText
 				local entryBox; entryBox = tabWindow:Add("Frame", {
 					Visible = false;
-					Size = UDim2.new(0, 200, 0, 75);
-					Position = UDim2.new(0.5, -100, 0.5, -100);
+					Size = UDim2.new(0, 241,0, 105);
+					BackgroundColor3 = Color3.fromRGB(45, 45, 45);
+					BorderColor3 = Color3.fromRGB(27, 42, 53);
+					Position = UDim2.new(0.434, -100,0.612, -100);
 					ZIndex = 100;
```
Comment (P3tray) [inline MainModule/Client/UI/Default/UserPanel.lua:199]: make it a bazillion for no good reason

File: MainModule/Client/UI/Default/UserPanel.lua [hunk 3]
```diff
@@ -243,13 +246,14 @@ return function(data, env)
 				})
 
 				entryText = entryBox:Add("TextBox", {
-					Position = UDim2.new(0, 55, 0, 10);
-					Size = UDim2.new(1, -60, 0, 25);
+					Position = UDim2.new(0.043, 0, 0.324, 0);
+					Size = UDim2.new(0.928, 0, 0.238, 0);
 					Text = "";
 					PlaceholderText = "Type entry here";
```
Comment (P3tray) [inline MainModule/Client/UI/Default/UserPanel.lua:252]: would this text be too confusing for the vast majority of roblox players?

**Review by P3tray — CHANGES_REQUESTED** [review]

File: MainModule/Client/UI/Default/UserPanel.lua (hunk 3 above)
Comment (EasternBloxxer) [reply MainModule/Client/UI/Default/UserPanel.lua:252]: I dont know. Not my Problem. Wont Fix,

File: MainModule/Client/UI/Default/UserPanel.lua (hunk 2 above)
Comment (EasternBloxxer) [reply MainModule/Client/UI/Default/UserPanel.lua:199]: yeah

**Pull request merged**